addon/components/_private/rdfa-editor/relationship-editor/modal.hbs

```diff
@@ -24,76 +20,51 @@
-              Add Type
+              Object type
             </AuLabel>
             <AuRadioGroup
               id={{id}}
               required={{true}}
-              name="types"
-              @selected={{@addRelationshipType}}
+              @name="object-type"
+              @selected={{this.selectedObjectType}}
               @width="block"
-              {{on "input" this.setAddType}}
+              @onChange={{this.setObjectType}}
               @alignment="inline"
               as |Group|
             >
-              {{#each this.types as |addType|}}
-                <Group.Radio
-                  name="types"
-                  value={{addType}}
-                >{{addType}}</Group.Radio>
+              {{#each this.objectTypes as |type|}}
+                <Group.Radio @value={{type}}>
+                  {{type}}
+                </Group.Radio>
               {{/each}}
             </AuRadioGroup>
-            {{#if this.errorMessage}}
-              <AuHelpText @error={{true}}>{{t
-                  "ember-rdfa-editor.image.url-error"
-                }}</AuHelpText>
-            {{/if}}
           {{/let}}
         </AuFormRow>
-        {{#if this.isAddExisting}}
-          <AuFormRow @alignment="default">
-            {{#let (unique-id) as |id|}}
-              <AuLabel for={{id}} @required={{true}} @requiredLabel="Required">
-                Object
-              </AuLabel>
-              <PowerSelect
-                id={{id}}
-                name="ids"
-                {{! For some reason need to manually set width }}
-                class="au-u-1-2"
-                @searchEnabled={{false}}
-                @options={{@rdfaIds}}
-                @selected={{this.objectRdfa}}
-                @onChange={{this.updateObject}}
-                as |obj|
-              >
-                {{obj.label}}
-              </PowerSelect>
-              {{#if this.errorMessage}}
-                <AuHelpText @error={{true}}>{{t
-                    "ember-rdfa-editor.image.url-error"
-                  }}</AuHelpText>
-              {{/if}}
-            {{/let}}
-          </AuFormRow>
-        {{/if}}
-        {{#if this.isAddResource}}
-          <AuFormRow @alignment="default">
-            {{#let (unique-id) as |id|}}
-              <AuLabel for={{id}} @required={{true}} @requiredLabel="Required">
-                URI base
-              </AuLabel>
-              <AuInput
-                id={{id}}
-                required={{true}}
-                value={{this.resourceUriBase}}
-                @width="block"
-                placeholder="http://example.com/resource/"
-                {{on "input" this.updateUriBase}}
-              />
-            {{/let}}
-          </AuFormRow>
-        {{/if}}
+        <AuFormRow @alignment="default">
+          {{#let (unique-id) as |id|}}
+            <AuLabel for={{id}} @required={{true}} @requiredLabel="Required">
+              Object
+            </AuLabel>
+            <PowerSelect
```

Comment:
When the PowerSelect is open, if I type anything in the console, I get an error
```
Assertion Failed: <PowerSelect> If you want the default filtering to work on options that are not plain strings, you need to provide `@searchField`
```
We should either configure filtering or I think we can disable it using `onInput`.

Reply:
Nice catch! I'll look into it.